<?xml version="1.0"?>

<launch>
  <arg name="robot_name" default="minicarcy" />
  <arg name="controller_type" default="drive" />
  <arg name="motors_controller_type" default="VelocityJointInterface" />

  <group ns="$(arg robot_name)">

    <!-- load config into parameter server -->
    <rosparam file="$(find minicarcy)/config/control_constants.yaml" command="load" />
    <param name="control_constants/controller_type" type="string" value="$(arg controller_type)" />
    <param name="control_constants/motors_controller_type" type="string" value="$(arg motors_controller_type)" />

    <!-- start ball tracking node-->
    <node name="minicarcy" pkg="minicarcy" type="mc_control_node" output="screen" />

  </group>
</launch>
